addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6205,6 +6213,38 @@ registry.ImageTools = ImageHandlerOption.extend({
         await this._applyShapeAndColors(true, newColors);
         img.classList.add('o_modified_image_to_save');
     },
+    /**
+     * Flip the image shape horizontally.
+     *
+     * @see this.selectClass for parameters
+     */
+    async setImgShapeFlipX(previewMode, widgetValue, params) {
```

Comment:
![image](https://github.com/odoo/odoo/assets/10338094/ef58c7a6-af91-41ad-af5f-9dadb78d8012)

Could you make it so all 4 buttons are the same width and their icon centered? :grimacing: Strange, is not that supposed to be automatic in the new design?

Reply:
It is due to a mix between `oi oi-fw` (2 first buttons) and `fa fa-fw` (2 last buttons).  I think it is something to be fixed by the designers.